site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,337 @@
+  ```java
+  ConcurrencyLimitingClient.newDecorator(new DynamicLimit());
+  
+  class DynamicLimit implements IntSupplier {
+
+    @Override
+    public int getAsInt() {
+      // Dynamically returns the current limitation.
+      ...
+    }
+  }
+  ```
+- You can now split an <type://HttpRequest> into <type://RequestHeaders>, `StreamMessage<HttpData>`
+  and trailers using <type://HttpRequest#split()>. #3924 #3953
+  ```java
+  HttpRequest request = HttpRequest.of(...);
+  SplitHttpRequest splitRequest = request.split();
+  
+  RequestHeaders headers = splitRequest.headers();
+  StreamMessage<HttpData> body = splitRequest.body();
+  CompletableFuture<HttpHeaders> trailers = splitRequest.trailers();
+  ```
+- Added more well-known MIME type constants to <type://MediaType>. #4040
+- You can now customize the <type://MediaType> of a file when using <type://FileService>. #4001 #4009
+  ```java
+  FileService.builder(resource)
+             .mediaTypeResolver((path, contentEncoding) -> {
+               if (path.endsWith(".proto")) {
+                 return MediaType.PROTOBUF;
+               }
+               if (path.endsWith(".csv")) {
+                 return MediaType.CSV_UTF_8;
+               }
+               return null;
+             })
+             .build();
+  ```
+- You can now send requests to a test server by using <type://ServerExtension#webClient()>
+  or <type://ServerRule#webClient()>. #3761 #3890
+  ```java
+   @RegisterExtension
+   static ServerExtension server = new ServerExtension() {
+     @Override
+     protected void configure(ServerBuilder sb) {
+       ...
+     }
+   };
+
+   @Test
+   void test() {
+     WebClient client = server.webClient(cb -> {  // 👈👈👈
+       cb.decorator(LoggingClient.newDecorator());
+     });
+     client.get("/foo").aggregate().join();
+   }
+  ```
+- You can now export metrics as [OpenMetrics](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md)
+  format using <type://PrometheusExpositionService>. #3926 #3928
+- You can now use <type://UnframedGrpcStatusMappingFunction> to customize how a gRPC status is mapped to 
+  an <type://HttpStatus> when using an unframed gRPC service. #3683 #3948
+  ```java
+  UnframedGrpcStatusMappingFunction mappingFunction = 
+      (ctx, grpcStatus, cause) -> {
+        if (grpcStatus.getCode() == DEADLINE_EXCEEDED) {
+            return INTERNAL_SERVER_ERROR;
+        }
+        // Pass to the default mapping function.
+        return null;
+      };
+  
+  UnframedGrpcErrorHandler errorHandler = 
+     UnframedGrpcErrorHandler.ofJson(mappingFunction);
+  
+  GrpcService
+      .builder()
+      .unframedGrpcErrorHandler(errorHandler);
+  ```
+- You can now expose `WebOperation`s of Actuator to a port of internal services. #3919 #3946
+  ```yaml
+  armeria:
+    ...
+  internal-services:
+    # Actuator will be exposed to 18080 port.
+    include: actuator, health 
+    port: 18080
```

Comment:
nit; I'm wondering if indentation is off here 

```suggestion
    internal-services:
      # Actuator will be exposed to 18080 port.
      include: actuator, health 
      port: 18080
```

Reply:
😱